profiles/kentik_snmp/cisco/cisco-nexus.yml

```diff
@@ -88,6 +88,9 @@ sysobjectid:
   - 1.3.6.1.4.1.9.12.3.1.3.2010    # NEXUS 93240YC-FX2
   - 1.3.6.1.4.1.9.12.3.1.3.2039    # NEXUS 3548P-XL
   - 1.3.6.1.4.1.9.12.3.1.3.2054    # MDS 9148T
+  - 1.3.6.1.4.1.9.12.3.1.3.2236    # Nexus 93600CDGX
+  - 1.3.6.1.4.1.9.12.3.1.3.2237    # Nexus 93360YCFX2
+  - 1.3.6.1.4.1.9.12.3.1.3.2238    # Nexus 93216TCFX2
```

Comment:
just noticed we're using different case on some of these, should we move them from `UPPER` to `Sentence`?